granary/tests/test_reddit.py

```diff
@@ -34,45 +37,31 @@ def __init__(self, fake_redditor):
 
 class RedditTest(testutil.TestCase):
 
-  def setUp(self):
-    super(RedditTest, self).setUp()
-    self.reddit = reddit.Reddit('token-here')
-
-  def test_user_to_actor(self):
-    actor = {
-      'objectType': 'person',
-      'displayName': 'bonkerfield',
-      'image': {'url': 'https://styles.redditmedia.com/t5_2az095/styles/profileIcon_ek6onop1xbf41.png?width=256&height=256&crop=256:256,smart&s=81fbf5a84c69533b5679f46638353d9e8471b817'},
-      'id': 'tag:reddit.com:bonkerfield',
-      'numeric_id': '59ucsixw',
-      'published': '2019-12-21T17:40:11Z',
-      'url': 'https://reddit.com/user/bonkerfield/',
-      'urls': [
-        {
-        "value":"https://reddit.com/user/bonkerfield/"
-        },
-        {
-        "value":"https://bonkerfield.org"
+  ACTIVITY = {
```

Comment:
minor: these go at module level, not class level

Reply:
oops, misunderstood, fixed.